ntpserver.py

```diff
@@ -307,6 +307,7 @@ def run(self):
 
 def do_ntp():
     print("NTP CODE IS RUNNING")
+    global stopFlag
```

Comment:
is this used?

Reply:
I believe that is needed to make sure it doesn't assign False to a new local variable called stopFlag when handling the keyboard interrupt. That line is included in other functions that reference stopFlag, such as WorkThread.run() and RecvThread.run().

Reply:
i totally missed it in the exception sorry.